src/Traits.h

```diff
@@ -207,7 +207,7 @@ template <typename T>
 struct TraitsImpl {
     static constexpr bool const             IsNullableType = false;
     static constexpr bool const             IsNativeNullableType = false;
-
+    static constexpr bool const             IsVectorType = false;

```

Comment:
>static constexpr bool const             IsVectorType = false; [](start = 4, length = 61)

This represents a slippery slope - if we add this,  should we add something that detects maps? Smart pointers? etc. This code should probably remain local to where it is used rather than putting it in Traits.h. 

If it needs to remain in Traits.h, there should be corresponding updates in Traits_UnitTest.cpp that validates the functionality.

Reply:
Input type is no longer a vector. IsVectorType removed